yt_dlp/extractor/nekohacker.py

```diff
@@ -0,0 +1,223 @@
+                    'track': '進め！むじなカンパニー (instrumental)',
+                    'track_number': 3,
+                    'duration': None
+                }
+            },
+            {
+                'url': 'https://nekohacker.com/wp-content/uploads/2021/01/むじな-de-なじむ-instrumental.mp3',
+                'md5': 'ebb0443039cf5f9ff7fd557ed9b23599',
+                'info_dict': {
+                    'id': '712',
+                    'ext': 'mp3',
+                    'title': 'むじな de なじむ (instrumental)',
+                    'thumbnail': 'https://nekohacker.com/wp-content/uploads/2021/01/OP表-1024x1024.png',
+                    'vcodec': 'none',
+                    'acodec': 'mp3',
+                    'release_date': '20210115',
+                    'album': '進め！むじなカンパニー',
+                    'artist': 'Nekohacker',
+                    'track': 'むじな de なじむ (instrumental)',
+                    'track_number': 4,
+                    'duration': None
+                }
+            }
+        ]
+    }]
+
+    def _real_extract(self, url):
+        playlist_id = self._match_id(url)
+
+        webpage = self._download_webpage(url, playlist_id)
+        playlist = get_element_by_class('playlist', webpage)
+
+        # handle cases which match _VALID_URL, but no element is found
+        if playlist is None:
+            try:
+                iframe = get_element_text_and_html_by_tag('iframe', webpage)
+            except ValueError:
+                iframe = None
+
+            # handle iframes instead of playlist
+            if iframe is not None:
+                attr = extract_attributes(iframe[1])
+                iframesrc = attr['src']
+                # handle spotify explicitly
+                if iframesrc is not None and re.match(r'https?://(?:www\.|open\.)?spotify\.com', iframesrc):
+                    raise ExtractorError('no playlist found, but iframe with spotify', expected=True)
+                raise ExtractorError('no playlist element found, but a unhandled iframe')
+
+            raise ExtractorError('no playlist element found - likely not a album', expected=True)
+
+        entries = []
+        for track_number, track in enumerate(re.findall(r'(<li[^>]+data-audiopath[^>]+>)', playlist), 1):
+            entry = traverse_obj(extract_attributes(track), {
+                'url': ('data-audiopath', {url_or_none}),
+                'ext': ('data-audiopath', {determine_ext}),
+                'id': 'data-trackid',
+                'title': 'data-tracktitle',
+                'track': 'data-tracktitle',
+                'album': 'data-albumtitle',
+                'duration': ('data-tracktime', {parse_duration}),
+                'release_date': ('data-releasedate', {lambda x: re.match(r'\d{8}', x.replace('.', ''))}, 0),
+                'thumbnail': ('data-albumart', {url_or_none}),
+            })
+            entry['track_number'] = track_number
+            entry['artist'] = 'Nekohacker'
```

Comment:
is the artist name actually "Neko Hacker"? just fixing my own previous suggestion
```suggestion
            entry['artist'] = 'Neko Hacker'
```

Reply:
i think so, if [youtube](https://www.youtube.com/channel/UCYYLjFslMa575OOJvV-h8xw) and [official site](https://nekohacker.com/) are to be believed